src/Symfony/Bridge/PhpUnit/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+4.1.0
+-----
+
+ * support for adding expected deprecations via `SymfonyTestsListenerTrait::expectDeprecation()`
```

Comment:
What about adding a new class to make this a bit more friendly,
eg `SymfonyTestHelper::expectDeprecation()` at the root of the bridge's namespace?

Reply:
Calling anything on a trait definitely smells.
Keeping it in sync with `$this->expectDeprecation()` called inside tests would be the best imho.